src/compat.jl

```diff
@@ -0,0 +1,7 @@
+if VERSION <= v"1.0.5"
```

Comment:
I notice there's a `backport-1.0` label in JuliaLang/julia#29442, I assume `oneunit(::CartesianIndex)` will be available?

Reply:
I added that backport label relatively recently, if memory serves. Hopefully that means it will be available; if not we can change the bounds here.